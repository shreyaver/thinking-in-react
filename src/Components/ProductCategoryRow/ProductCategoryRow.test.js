import React from 'react';
import renderer from 'react-test-renderer';
import ProductCategoryRow from '.';

describe('ProductCategoryRow', () => {
  it('renders without crashing', () => {
    const productCategoryRow = renderer.create(<ProductCategoryRow category="Electronics" />).toJSON();
    expect(productCategoryRow).toMatchSnapshot();
  });
});
