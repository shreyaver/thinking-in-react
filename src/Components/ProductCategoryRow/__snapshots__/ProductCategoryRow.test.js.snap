// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCategoryRow renders without crashing 1`] = `
<tr>
  <th
    colSpan="2"
  >
    Electronics
  </th>
</tr>
`;
